atef/config.py

```diff
@@ -686,9 +692,29 @@ def walk_comparisons(self) -> Generator[PreparedComparison, None, None]:
         """Walk through the prepared comparisons."""
         yield from self.comparisons
 
+    def get_status_logger(self) -> logging.Logger:
+        """
+        Get the status logger for this step.
+        """
+        top_file = get_parent_file(self)
+        file_id = getattr(top_file, "uuid", "status_logger")
+        if isinstance(file_id, UUID):
+            return configure_and_get_status_logger(file_id)
+        else:
+            return logging.getLogger(file_id)
+
     async def compare(self) -> Result:
         """Run all comparisons and return a combined result."""
+        status_logger = self.get_status_logger()
         results = []
+        try:
+            cfg_name = attrgetter("config.name")(self)
```

Comment:
Question: why is attrgetter used here instead of:
```suggestion
            cfg_name = self.config.name
```